README.md

```diff
@@ -6,118 +6,131 @@
-    7. You now have a repository in your own GitHub account that is a copy of the Managed Structure Template for you to begin working with.
+   1. Make sure you're logged in to GitHub.
+   2. Go to the [Managed Structure Template repo](https://github.com/griptape-ai/managed-structure-template).
+   3. Select the _Use this template_ drop-down.
+   4. Choose _Create a new repository_.
+   5. Provide a name for the new repository and (optionally) a description.
+   6. Press the _Create repository_ button.
+   7. You now have a repository in your own GitHub account that is a copy of the Managed Structure Template for you to begin working with.
 2. Clone your newly-created repository to a directory in your local development environment so that you can begin authoring your own Griptape Cloud Managed Structure.
 3. Start Skatepark.
-    ```bash
-    gt skatepark start
-    ```
+   ```bash
+   gt skatepark start
+   ```
 4. Open a new terminal window. Navigate to the directory with the locally-cloned repository.
 5. Register a Structure with Skatepark.
-    ```bash
-    gt skatepark register --main-file structure.py
-    ```
-    This will result in the ID of the registered Structure. It is important to make note of this ID as it will be used to distinguish which Structure you want to run. You can register any number of Structures.
 
-    The example client program uses the environment variable `GT_STRUCTURE_ID` to determine which Structure to run.
-    Set this environment variable to the Structure ID you registered in the previous step.
-    ```bash
-    export GT_STRUCTURE_ID={STRUCTURE_ID}
-    ```
+   ```bash
+   gt skatepark register --structure-config-file structure_config.yaml
+   ```
 
-    Or you can register and set the environment variable in one step.
-    ```bash
-    export GT_STRUCTURE_ID=$(gt skatepark register --main-file structure.py --tldr)
-    ```
+   This will result in the ID of the registered Structure. It is important to make note of this ID as it will be used to distinguish which Structure you want to run. You can register any number of Structures.
+
+   The example client program uses the environment variable `GT_STRUCTURE_ID` to determine which Structure to run.
+   Set this environment variable to the Structure ID you registered in the previous step.
+
+   ```bash
+   export GT_STRUCTURE_ID={STRUCTURE_ID}
+   ```
+
+   Or you can register and set the environment variable in one step.
+
+   ```bash
+   export GT_STRUCTURE_ID=$(gt skatepark register --structure-config-file structure_config.yaml --tldr)
+   ```
 
 > [!IMPORTANT]
-> Structures registered with the Skatepark are not persisted across restarts. You will need to re-register the Structure each time you restart Skatepark.
-6. Confirm that the Structure is registered.
+> Structures registered with the Skatepark are not persisted across restarts. You will need to re-register the Structure each time you restart Skatepark. 6. Confirm that the Structure is registered.
```

Comment:
does this need a newline before the `6.`?

Reply:
Yes! Fixed this